topology/University of Michigan/AGLT2/AGLT2_downtime.yaml

```diff
@@ -865,3 +865,58 @@
   Services:
   - GridFtp
 # ---------------------------------------------------------
+- Class: SCHEDULED
+  ID: 880780788
+  Description: server room recabling
+  Severity: Outage
+  StartTime: Jun 14, 2021 05:00 +0000
+  EndTime: Jun 18, 2021 23:00 +0000
+  CreatedTime: Jun 07, 2021 15:01 +0000
+  ResourceName: AGLT2_SL7
+  Services:
+  - CE
+# ---------------------------------------------------------
+- Class: SCHEDULED
+  ID: 880781799
+  Description: server room recabling
+  Severity: Outage
+  StartTime: Jun 14, 2021 05:00 +0000
+  EndTime: Jun 18, 2021 23:00 +0000
+  CreatedTime: Jun 07, 2021 15:02 +0000
+  ResourceName: AGLT2_CE_2
+  Services:
+  - CE
+# ---------------------------------------------------------
+- Class: SCHEDULED
+  ID: 880782188
+  Description: server room recabling
+  Severity: Outage
+  StartTime: Jun 14, 2021 05:00 +0000
+  EndTime: Jun 18, 2021 23:00 +0000
+  CreatedTime: Jun 07, 2021 15:03 +0000
+  ResourceName: AGLT2_TEST_CE
+  Services:
+  - CE
```

Comment:
```suggestion
```

The test CE downtime should go into https://github.com/opensciencegrid/topology/blob/master/topology/University%20of%20Michigan/AGLT2/AGLT2_TEST_downtime.yaml

Reply:
Hi Brian. 
Thanks for the fix..
Next time we will exclude the Test CE from this file